docs/rules/no-js-extension.md

```diff
@@ -36,6 +43,55 @@ define(function (require) {
     var foo = require('foo');
     /* ... */
 });
+
+require(['pluginname!foo'], function (foo) {
```

Comment:
The default behavior is: _ignore plugin-prefixed paths_
If you add some plugin prefix to options array, then: _ignore plugin-prefixed paths, unless they start with some of the listed plugin prefixes_